cmd/srcd/cmd/prune_test.go

```diff
@@ -100,6 +104,69 @@ func (s *PruneTestSuite) TestRunningContainers() {
 	require.Equal(netNames(prevNets), netNames(nets))
 }
 
+type PruneWithoutImagesTestSuite struct {
+	PruneTestSuite
+}
+
+func (s *PruneWithoutImagesTestSuite) TestRunningContainers() {
+	s.testRunningContainers(s.Require(), false)
+}
+
+func (s *PruneWithoutImagesTestSuite) TestStoppedContainers() {
+	require := s.Require()
+
+	out, err := s.RunCommand(context.TODO(), "prune")
+	require.NoError(err, out.String())
+}
+
+type PruneWithImagesTestSuite struct {
+	PruneTestSuite
+}
+
+func (s *PruneWithImagesTestSuite) TestRunningContainersWithImages() {
+	if os.Getenv("TEST_PRUNE_WITH_IMAGE") != "true" {
+		s.T().Skip("Skipping testing pruning with --with-images flag")
```

Comment:
Suggestion to make it clear how to run without having to dig into the code:
```suggestion
		s.T().Skip("Use env var TEST_PRUNE_WITH_IMAGE=true to test srcd prune with --with-images flag")
```